types/react-dom/experimental.d.ts

```diff
@@ -34,3 +34,12 @@ export {};
 
 declare module "." {
 }
+
+declare module "react" {
+    interface ViewTransitionInstance {
+        group: Animatable;
+        imagePair: Animatable;
+        old: Animatable;
+        new: Animatable;
```

Comment:
It's handwaving that React actually implements this. So technically it could drift when the DOM interface gets extended and React hasn't caught up. 

But this way people discover new properties quicker and report to us if it doesn't work.